vic.api.v0.yaml

```
@@ -937,6 +937,59 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/Error"
+  "/invoice/{id}/reject":
+    post:
+      description: |
+        Used to communicate that the invoice data have NOT been
+        successfully posted to the ERP, due to invalid data. This should NOT
```

Let's add some bold emphases on **NOT**

```suggestion
        Used to communicate that the invoice data have **NOT** been
        successfully posted to the ERP, due to invalid data. This should **NOT**
```